pass-data-client/src/main/java/org/eclipse/pass/support/client/model/Policy.java

```diff
@@ -160,7 +160,7 @@ public List<Repository> getRepositories() {
      * @param repositories list repositories to set
      */
     public void setRepositories(List<Repository> repositories) {
-        this.repositories = repositories;
+        this.repositories = repositories == null ? new ArrayList<>() : repositories;
```

Comment:
What do you think about extracting the ternary out into a helper method named something like `getListNullCheck`?